Extensions/3D/CustomRuntimeObject3DRenderer.ts

```diff
@@ -0,0 +1,132 @@
+namespace gdjs {
+  export class CustomRuntimeObject3DRenderer
```

Comment:
This class is mostly straightforward and what I would expect from something that wraps a Three.js group. Nice.